pygen/pygen_src/riscv_asm_program_gen.py

```diff
@@ -596,17 +596,17 @@ def get_directed_instr_stream(self):
             arg = "directed_instr_{}".format(i)
             stream_name_opts = "stream_name_{}".format(i)
             stream_freq_opts = "stream_freq_{}".format(i)
-            if(arg in args):
-                val = args_dict[arg]
+            if(arg in cfg.argv):
+                val = cfg.args_dict[arg]
                 opts = val.split(",")
                 if(len(opts) != 2):
                     logging.critical(
                         "Incorrect directed instruction format : %0s, expect: name,ratio", val)
                 else:
                     self.add_directed_instr_stream(opts[0], int(opts[1]))
-            elif(stream_name_opts in args and stream_freq_opts in args):
-                stream_name = args_dict[stream_name_opts]
-                stream_freq = args_dict[stream_freq_opts]
+            elif(stream_name_opts in cfg.argv and stream_freq_opts in cfg.argv):
```

Comment:
same as above, should be `elif(stream_name_opts in cfg.args_dict and stream_freq_opts in cfg.args_dict):`

Reply:
will fix this one